### PR TITLE
[CI:DOCS] update basic_networking.md

### PR DESCRIPTION
**Pull request by RonNair24**
added information regarding opening host firewall ports for rootless containers.

#### Does this PR introduce a user-facing change?

```release-note
None

```

### CODE REVIEW

File: docs/tutorials/basic_networking.md [hunk 1]
```diff
@@ -257,6 +257,15 @@ valid_lft 78808sec preferred_lft 78808sec
 inet6 fe80::5632:6f10:9e76:c33/64 scope link noprefixroute
 valid_lft forever preferred_lft forever
 ```
+
+For rootless containers, make sure to open the firewall ports on the host so that a client outside the host can communicate.
+An example if you are using a red hat based distro.
```
Comment (Luap99) [inline docs/tutorials/basic_networking.md:262]:
```suggestion
An example if you are using firewalld:
```

**Review by Luap99 — COMMENTED** [review]
Thanks, one suggestion. Also you have a trailing white space somewhere. You need to remove it to pass CI checks.

**Review by RonNair24 — COMMENTED** [review]
Done

**Comment by TomSweeneyRedHat** [comment]
LGTM, but you still have space issues.  Perhaps `sed -i 's/\s*$//g' docs/tutorials/basic_networking.md` and then resubmit?

**Review by vrothberg — COMMENTED** [review]
@RonNair24 friendly ping.

Can you also squash the two commits into one before repushing?

**Review by RonNair24 — COMMENTED** [review]
Done

**Comment by vrothberg** [comment]
I still see three commits https://github.com/containers/podman/pull/17792/commits.

@RonNair24, can you squash and repush?

**Comment by rhatdan** [comment]
@RonNair24 Are you still working on this?

**Comment by vrothberg** [comment]
Going through old inactive PRs...
Feel free to re-open when reviving it.

**Pull request closed without merging**